docs/installation/sources/sources_linux.rst

```diff
@@ -0,0 +1,210 @@
+
+    Where :code:`PATH_TO_WORKSPACE` is the path to the :code:`Fast-DDS-ws` worskspace.
+
+.. note::
+
+    If you want to compile the examples, you will need to add the flag
+    :code:`--cmake-args "-DCOMPILE_EXAMPLES=ON"` when running :code:`colcon build`.
+    If you want to compile the performance tests, you will need to add the flag
+    :code:`--cmake-args "--DPERFORMANCE_TESTS=ON"` when running :code:`colcon build`.
+    For this step, you need Gtest_ as explained in the requirements_linux_sources_ section above.
+
+
+.. _cmake_installation_linux:
+
+CMake Installation
+------------------
+
+This section explains how to compile *eProsima Fast DDS* locally with CMake_.
+First of all, create a Fast-DDS directory where to download and build *eProsima Fast DDS* and its dependencies:
+
+.. code-block:: bash
+
+    mkdir Fast-DDS && cd Fast-DDS
+
+Now clone the following dependencies and compile them using CMake_.
+
+* `Foonathan memory <https://github.com/foonathan/memory>`_
+
+  .. code-block:: bash
+
+      $ git clone https://github.com/eProsima/foonathan_memory_vendor.git
+      $ mkdir foonathan_memory_vendor/build && cd foonathan_memory_vendor/build
+      $ cmake .. -DCMAKE_INSTALL_PREFIX=../../install
+      $ sudo cmake --build . --target install
+      $ cd ../..
+
+* `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+  .. code-block:: bash
+
+      $ git clone https://github.com/eProsima/Fast-CDR.git
+      $ mkdir Fast-CDR/build && cd Fast-CDR/build
+      $ cmake .. -DCMAKE_INSTALL_PREFIX=../../install
+      $ sudo cmake --build . --target install
+      $ cd ../..
+
+Once all dependencies are installed, you will be able to compile and install *eProsima Fast DDS*:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/eProsima/Fast-RTPS.git
+    $ mkdir Fast-RTPS/build && cd Fast-RTPS/build
+    $ cmake ..  -DCMAKE_INSTALL_PREFIX=../../install -DCMAKE_PREFIX_PATH=../../install
+    $ sudo cmake --build . --target install
+
+
+
+If you want to install *eProsima Fast DDS* system-wide instead of locally, you need to remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR` and :code:`Fast-RTPS`, and change the one in the
+configuration step of :code:`foonathan_memory_vendor` to the following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=/usr/local/ -DBUILD_SHARED_LIBS=ON
+
+.. note::
+
+    If you want to compile the examples, you will need to add the argument :code:`-DCOMPILE_EXAMPLES=ON` when calling
+    the configuration CMake_.
+    If you want to compile the performance tests, you will need to add the argument
+    :code:`-DPERFORMANCE_TESTS=ON` when calling the configuration CMake_.
+    For this step, you need Gtest_ as explained in the requirements_linux_sources_ section above.
+
+.. important::
+
+    When running an *eProsima Fast DDS* application, you need to link it with the library :code:`/usr/local/lib/`
```

Comment:
I think this sentence is weird; the library is not `/usr/local/lib/`, that is merely the directory where the library is.

Reply:
ok. changed from:
the library :code:`/usr/local/lib/` 
to:
the library in :code:`/usr/local/lib/`